.github/workflows/firebase-deploy-ios.yml

```diff
@@ -0,0 +1,74 @@
+name: iOS Development Build
```

Comment:
@camerow you can ignore this action. It's just doing a direct deploy to `firebase`

They could be useful as a reference for `runs-on: macos-latest` etc. as this works with doing `eas build` then deploy to firebase